### PR TITLE
chore: add context to serverless and database driver adapters

### PR DESCRIPTION
**Pull request by ruheni**

### CODE REVIEW

File: content/200-concepts/100-components/database-drivers.mdx [hunk 1]
```diff
@@ -17,15 +17,21 @@ Prisma Client can connect and run queries against your database using JavaScript
 
 Prisma will use the Query Engine to transform the Prisma Client query to SQL and run the generated SQL queries via the JavaScript database driver.
 
-![Query flow from the user application to the database using Prisma Client and driver adapters](./images//drivers/qe-query-engine-adapter.png)
+![Query flow from the user application to the database using Prisma Client and driver adapters](./images/drivers/qe-query-engine-adapter.png)
```
Comment (janpio) [inline content/200-concepts/100-components/database-drivers.mdx:21]: Here (line 21) would be a good spot to introduce that there are different kind of driver adapters maybe.

File: content/200-concepts/100-components/database-drivers.mdx [hunk 2]
```diff
@@ -17,15 +17,21 @@ Prisma Client can connect and run queries against your database using JavaScript
...
 ### Serverless driver adapters
 
+Serverless driver adapters allow you to connect to your database using other protocols, such as HTTP and WebSockets. Prisma maintains the following serverless driver adapters:
+
 - [Neon](/guides/database/neon#how-to-use-the-neon-serverless-driver-with-prisma-preview)
 - [PlanetScale](/guides/database/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-preview)
 
 ### Database driver adapters
 
+Database driver adapters enable you to connect to your database using the default protocol that a database provider offers. For example, Turso only supports HTTP connections.
```
Comment (janpio) [inline content/200-concepts/100-components/database-drivers.mdx]: I think the database driver adapters are less about the protocol, but more about that we just want to support these Node based drivers.

In the future we will also list `pg` and `Postgre.js` here for example.

File: content/200-concepts/100-components/database-drivers.mdx [hunk 3]
```diff
@@ -17,17 +17,28 @@ Prisma Client can connect and run queries against your database using JavaScript
...
 
 ### Database driver adapters
 
+You can connect to your database using a Node.js-based driver from Prisma Client using a database driver adapter. Prisma maintains the following database driver adapters:
+
 - [Turso](/guides/database/turso#connect-and-query-your-primary-database)
 
+### Serverless driver adapters
+
+Database providers, such as Neon and PlanetScale, allow you to connect to your database using other protocols, such as HTTP and WebSockets. These database drivers are optimized for connecting to your database in serverless and edge environments.
```
Comment (petradonka) [inline content/200-concepts/100-components/database-drivers.mdx]: "other protocols" - other than TCP, other than database driver adapters do? Might be worth spelling out what the baseline is that we're referring to?

File: content/200-concepts/100-components/database-drivers.mdx [hunk 4]
```diff
@@ -17,17 +17,28 @@ Prisma Client can connect and run queries against your database using JavaScript
...
+
+Prisma maintains the following serverless driver adapters:
+
+- [Neon](/guides/database/neon#how-to-use-the-neon-serverless-driver-with-prisma-preview)
+- [PlanetScale](/guides/database/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-preview)
+
 ## Community maintained database driver adapters
```
Comment (janpio) [inline content/200-concepts/100-components/database-drivers.mdx:43]: While we are making it more elaborate, let's also add an explanation here. Something that says that anyone can create a driver adapter and some members of our community have done so.

Comment (ruheni) [reply content/200-concepts/100-components/database-drivers.mdx:43]: How soon do we intend to create a "specification" for community members to build their own driver adapters? I think that detail may also be helpful for those looking to do that. Alternatively, we can ask them to reach out if they're interested, but I'm currently unsure where to point them.

It's a small note that would probably add some value to the community and the docs.

Comment (janpio) [reply content/200-concepts/100-components/database-drivers.mdx:43]: We do not have an ETA for that.

**Review by janpio — APPROVED** [review]

**Pull request merged**